ui/src/main/resources/ui/images/kettle_logo_small.svg

```diff
@@ -1,16 +1,71 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
-<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="16px" height="16px" viewBox="0 0 16 16" enable-background="new 0 0 16 16" xml:space="preserve">  <image id="image0" width="16" height="16" x="0" y="0"
-    href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAMAAAAoLQ9TAAAABGdBTUEAALGPC/xhBQAAACBjSFJN
-AAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAAAvVBMVEXMAADLAADMAwPMBATM
-AQHMAgLIAADKAADJAADfamrnlJTRIyPWOTnfaWnnmprbWFjrrKzTJibww8PwxMTnj4/bWlrHAADs
-srLQHR3qp6fqqKjss7PRHx/stLTstbXz2dnPGRnidXX03d3RISHMCwvODQ3TKyv03t7PHh7QGxvT
-LCzqq6vrq6vifHzwx8fWODjNDQ3y19f03NzQHBzSLCzRHR302trSJCTNDg7OExPSJibRICDbVlbR
-IiL///+UQJfzAAAAAWJLR0Q+SWQA4wAAAAd0SU1FB+MFDhU2O1bBIgUAAAC/SURBVBjTTY/nDsIw
-DISdJrEDHYRZVoEyStl7w/u/FkkQAsl/7pPuzgfAwOOCSckE94ywh0igFBCik8AKRR+DAP1iwUoP
-w6iky5VqWZeiED3gWKs34mar3ew06jXkIKib9PqDNB0MYz0iYVJoPMmmWg/z2RyFrRFS+Qui5aq9
-3ihpWwxY/gHj2QY7vdjnh6OzCDrF5+A0yvodF2rAZR1flbr1PrXWcn+QBJ483WOOkOmi1+d1S4TJ
-Yr9x8MVu/htUqw4FKPWaMAAAACV0RVh0ZGF0ZTpjcmVhdGUAMjAxOS0wNS0xNVQwNDo1NDo1OS0w
-NzowMDmUcF0AAAAldEVYdGRhdGU6bW9kaWZ5ADIwMTktMDUtMTVUMDQ6NTQ6NTktMDc6MDBIycjh
-AAAAAElFTkSuQmCC" />
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
```

Comment:
Don't you want to remove this? Just in case

Reply:
I think there is no problem (https://inkscape.org/pt/acerca-de/licenciamento/) but anyway.